src/base/utils/net.cpp

```diff
@@ -38,5 +38,27 @@ namespace Utils
         {
             return !QHostAddress(ip).isNull();
         }
+
+        bool isLoopbackAddress(const QHostAddress &addr)
+        {
+            return addr == QHostAddress::LocalHost
+                    || addr == QHostAddress::LocalHostIPv6
+                    || addr == QHostAddress(QLatin1String("::ffff:127.0.0.1"));
+        }
+
+        bool isLANAddress(const QHostAddress &addr)
+        {
+            if (addr.isInSubnet(QHostAddress(QLatin1String("0.0.0.0")), 0)) {
```

Comment:
we use implicit type conversion in our codebase, so I'll say there is no need for explicit `QHostAddress()` construction here.